packages/react/src/components/ComboBox/ComboBox-story.js

```diff
@@ -39,7 +39,7 @@ const items = [
 
 const sizes = {
   'Extra large size (xl)': 'xl',
-  'Regular size (lg)': '',
+  'Regular size (lg)': undefined,
```

Comment:
I realize this is kinda a requirement of the `select` knob, but having "Regular size (lg)" like this makes it seem like `lg` is a valid value (it confused me yesterday, for example.)

Reply:
Good point, I'll remove that as well